packages/tictactoe/package.json

```diff
@@ -106,9 +106,9 @@
     "build": "node --max-old-space-size=4076 scripts/build.js",
     "test": "run-s test:truffle 'test:app --all'",
     "test:app": "node scripts/test.js --env=jsdom",
-    "test:truffle": "node scripts/test_truffle.js",
+    "test:contracts": "node scripts/test_truffle.js",
```

Comment:
Perhaps rename `test_truffle.js` to `test_contracts.js`?

Reply:
Have now refactored the tests using jest.

Reply:
This has been addressed as follows: all packages use `test:integration` and use jest (not truffle).